src/main/java/org/jboss/remoting3/ClientServiceHandle.java

```diff
@@ -141,6 +141,6 @@ public void handleDone(final T result, final FutureResult<T> futureResult) {
      * Close the channel associated with this handle
      */
     public void closeChannel() throws IOException {
-        channel.close();
+        safeClose(channel);
```

Comment:
Thanks @ropalka I was just going to ask about this as I see if when running Jakarta EE 9.1 Signature testing (not sure if it has an impact)